web/html/src/manager/state/recurring-states-list.js

```diff
@@ -116,25 +62,86 @@ class RecurringStatesList extends React.Component {
+                                    cell={(row, criteria) => row.cron}
+                                />
+                                { this.props.disableCreate ?
+                                    <Column
+                                        columnClass="text-center"
+                                        headerClass="text-center"
+                                        columnKey="targetType"
+                                        header={t('Target Type')}
+                                        cell={(row, criteria) => targetTypeToString(row.targetType)}
+                                    />
+                                    : null }
+                                <Column
+                                    columnClass="text-right"
+                                    headerClass="text-right"
+                                    header={t('Actions')}
+                                    cell={(row, criteria) => {
+                                        return (
+                                            <div className="btn-group">
+                                                <Button
+                                                    className="btn-default btn-sm"
+                                                    title={t("Details")}
+                                                    icon="fa-list"
+                                                    handler={() => {this.props.onSelect(row)}}
+                                                />
+                                                <Button
+                                                    className="btn-default btn-sm"
+                                                    title={t("Edit")}
+                                                    icon="fa-edit"
+                                                    handler={() => {this.props.onEdit(row)}}
+                                                />
+                                                <ModalButton
+                                                    className="btn-default btn-sm"
+                                                    title={t("Delete")}
+                                                    icon="fa-trash"
+                                                    target="delete-modal"
+                                                    item={row}
+                                                    onClick={i => this.selectToDelete(i)}
+                                                />
+                                            </div>
+                                        );
+                                    }}
+                                />
+                            </Table>
+                            <DeleteDialog id="delete-modal"
+                                          title={t("Delete Recurring State Schedule")}
+                                          content={<span>{t("Are you sure you want to delete the selected item?")}</span>}
```

Comment:
I don't think we need a `span` wrapping here as it end up being used [here](https://github.com/uyuni-project/uyuni/blob/master/web/html/src/components/popup.js#L44)